backend/api/hotels/types.py

```diff
@@ -0,0 +1,12 @@
+import strawberry
+
+from ..helpers.i18n import make_localized_resolver
+
+
+@strawberry.type
+class HotelRoom:
+    id: str
+    name: str = strawberry.field(resolver=make_localized_resolver("name"))
+    description: str = strawberry.field(resolver=make_localized_resolver("description"))
+    price: int
+    is_sold_out: bool
```

Comment:
I'd do a quantity left, so we prevent to book more rooms than are available, but this can be dealt with later (we are not going to have a lot of people booking for more than one room)

Reply:
hmm I see, it makes sense to block if the user selects like 30 rooms and we have only 25. I will add a check that the number of rooms you are buying is less than the available quantity

Reply:
I added a check so that the user cannot buy  200 rooms in one go when the capacity is 50

Reply:
done frontend check